packages/edit-site/src/store/actions.js

```diff
@@ -43,7 +43,19 @@ export function __experimentalSetPreviewDeviceType( deviceType ) {
  *
  * @return {Object} Action object.
  */
-export function setTemplate( templateId ) {
+export function* setTemplate( templateId ) {
+	if ( Number.isNaN( Number( templateId ) ) ) {
```

Comment:
Out of curiosity, why do we need such involved check? 😮 
What kind of data could we expect here?
The function docs say that `templateId` is supposed to be a number, so... couldn't we just go with a simple falsy check?

Reply:
Replaced check with a simple `typeof`. Updates the doc which explains what the function now does.